### ISSUE TITLE
[ReferenceError] "ReferenceError: window is not defined" on adaptivecards@2.11.0

### ISSUE THREAD

**Issue by swatDong** [microsoft/AdaptiveCards#7904]
### Target Platforms

NodeJS

### SDK Version

2.11.0

### Application Name

Node console app

### Problem Description

My project is a node backend app using `adaptivecards` to parse/render/verify card content on server side.
It was using `adaptivecards@2.10.0` and everything worked well. After switching to the latest `adaptivecards@2.11.0`, it throws **ReferenceError: window is not defined** on `require("adaptivecards")`.

ReferenceError: window is not defined
    at Object.<anonymous> (...\node_modules\adaptivecards\lib\utils.js:149:26)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (...\node_modules\adaptivecards\lib\adaptivecards.js:22:14)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)

### Screenshots

<img width="874" alt="image" src="https://user-images.githubusercontent.com/7642967/190854152-57ec7295-e774-4bc6-9f7b-e3b9c3f6c69e.png">

### Card JSON

```JSON
{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "type": "AdaptiveCard",
  "version": "1.4",
  "body": [
  ]
}
```

### Sample Code Language

javascript

### Sample Code

const { AdaptiveCard } = require("adaptivecards");  // error occurs at this line

const card = require("./card.json");
const ac = new AdaptiveCard();
ac.parse(card);

**Comment by dupontbenoit** [comment]
Same for me when I try to run this sample repo code.

https://github.com/OfficeDev/TeamsFx-Samples/tree/v1.0.0/bot-sso

**Comment by paulcam206** [comment]
Hey @swatDong - this has been fixed in [`2.11.1`](https://www.npmjs.com/package/adaptivecards/v/2.11.1)! Please let us know if you encounter any further difficulties.
